src/Atlas.cc

```diff
@@ -352,6 +354,17 @@ void Atlas::PostLoad()
     mvpBackupMaps.clear();
 }
 
+void Atlas::SaveAtlas(string pathSaveFileName, string strVocabularyName, string strVocabularyChecksum) {
+    unique_lock<mutex> lock(mMutexAtlas);
+    this->PreSave();
+    cout << pathSaveFileName << endl;
+    std::ofstream ofs(pathSaveFileName, std::ios::binary);
+    boost::archive::binary_oarchive oa(ofs);
+    oa << strVocabularyName;
+    oa << strVocabularyChecksum;
+    oa << this;
```

Comment:
[major] I expect we need to close the stream after writing out the atlas.

Reply:
Looking at examples it seems that it closes the stream when it goes out of scope/